BuildAutomation/BuildOpenvSwitch.ps1

```diff
@@ -1,7 +1,7 @@
-Param(
-  [string]$OVSGitBranch = "branch-2.5-cloudbase",
-  [string]$SignX509Thumbprint
-)
+Param(
+  [string]$OVSGitBranch = "branch-2.5-cloudbase",
```

Comment:
we could always revert commit: https://github.com/cloudbase/openvswitch-hyperv-installer/commit/4d2bbd24646cc1f73edb0e586defecc621e45d00 (hich introduced them) and fix white-spaces and do another pull request.